html/semantics/scripting-1/the-script-element/script-charset-02.html

```diff
@@ -23,7 +23,7 @@
   <script>
   test(function() {
     //these strings should not match, since the tested file is in windows-1250, and fallback is defined as utf-8
-    assert_not_equals(window.getSomeString().length, 5);
+    assert_not_equals(window.getSomeString().length, 10);
```

Comment:
Rather than checking the string length, could you check the exact value? `assert_equals(window.getSomeString(), "\u015b\u0107\u0105\u017c\u017a");` in script-charset-03.html is an example.